server/src/main/java/eu/kanade/tachiyomi/network/interceptor/DriverJar.java

```diff
@@ -26,22 +26,9 @@
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 
-/*
-  exact copy of https://github.com/microsoft/playwright-java/blob/4d278c391e3c50738ddea6c3e324a4bbbf719d86/driver-bundle/src/main/java/com/microsoft/playwright/impl/driver/jar/DriverJar.java
-  with diff:
-  108a109,116
-  >
-  >   private FileSystem initFileSystem(URI uri) throws IOException {
-  >     try {
-  >       return FileSystems.newFileSystem(uri, Collections.emptyMap());
-  >     } catch (FileSystemAlreadyExistsException e) {
-  >       return null;
-  >     }
-  >   }
-  116c124
-  <     try (FileSystem fileSystem = "jar".equals(uri.getScheme()) ? FileSystems.newFileSystem(uri, Collections.emptyMap()) : null) {
-  ---
-  >     try (FileSystem fileSystem = "jar".equals(uri.getScheme()) ? initFileSystem(uri) : null) {
+/**
```

Comment:
can we use be a better approach? i.e. running `Chromium.preinstall` in serverSetup
I prefer to not have a custom DriverJar for playwright

Reply:
We cannot, we need a custom DriverJar so that we make it only install chromium(line 96)